pkg/vpcmodel/drawioOutput.go

```diff
@@ -18,35 +19,31 @@ func (e *edgeInfo) IsExternal() bool {
 }
 
 // DrawioOutputFormatter create the drawio connectivity map.
-// It build the drawio tree out of the VPCConfig and VPCConnectivity, and output it to a drawio file
+// It builds the drawio tree out of the VPCConfigs and VPCConnectivitys, and outputs it to a drawio file
 // the steps of creating the drawio tree:
 // 1. collect all the connectivity edges to a map of (src,dst,label) -> isDirected. also mark the nodes that has connections
 // 2. create the treeNodes of the NodeSets, filters. routers and nodes
-// 3. create the edges from the map we created in stage (1). also also set the routers to the edges
+// 3. create the edges from the map we created in stage (1). also sets the routers to the edges
 
 type DrawioOutputFormatter struct {
-	cConfig *VPCConfig
-	conn    *GroupConnLines
-	gen     *DrawioGenerator
-	routers map[drawio.TreeNodeInterface]drawio.IconTreeNodeInterface
-	uc      OutputUseCase
+	cConfigs map[string]*VPCConfig
+	conns    map[string]*GroupConnLines
+	gen      *DrawioGenerator
+	routers  map[drawio.TreeNodeInterface]drawio.IconTreeNodeInterface
+	uc       OutputUseCase
 }
 
-func (d *DrawioOutputFormatter) init(cConfig *VPCConfig, conn *GroupConnLines, uc OutputUseCase) {
-	d.cConfig = cConfig
-	d.conn = conn
+func (d *DrawioOutputFormatter) init(cConfigs map[string]*VPCConfig, conns map[string]*GroupConnLines, uc OutputUseCase) {
+	d.cConfigs = cConfigs
+	d.conns = conns
 	d.uc = uc
-	d.gen = NewDrawioGenerator(cConfig.CloudName)
+	// just take the cloud name from one of the configs
+	_, aVpcConfig := common.AnyMapEntry(cConfigs)
+	cloudName := aVpcConfig.CloudName
+	d.gen = NewDrawioGenerator(cloudName)
 	d.routers = map[drawio.TreeNodeInterface]drawio.IconTreeNodeInterface{}
```

Comment:
I am not sure I understand this comment.
the cloud name here is the name of the ibm cloud, not the name of the vpc.
currently, all VPCConfig.CloudName is "IBM Cloud"
my drawio code currently supporting only one cloud.
I am using anyMapEntry() just to get the name of the cloud, not the name of the vpc.
if we will support multi clouds, I will change this code to iterate over a list of clouds, and there will not be need for anyMapEntry()

Reply:
I think it might help to have specific methods for a type `multipleVPCConfigs`  (e.g. `getCloudNames()` ) so that the functionality is more clear (internally they can call `AnyMapEntry`)

Reply:
maybe open a separate issue for this, to consider if other functionality is required for such a type.

Reply:
there too many uses for map[string]*vpcmodel.VPCConfig or map[string]*VPCConfig,
moving it to separate issue